src/main/java/core/basesyntax/RightTriangleSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RightTriangleSupplier {
+    private Random random = new Random();
+    private RandomSize size = new RandomSize();
+    private ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomRightTriangle() {
+        return new RightTriangle(size.getRandomSize(),
+                size.getRandomSize(), color.getRandomColor());
+    }
+}

```

Comment:
According to the checklist, we should not begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition (line 5) and the `getRandomRightTriangle` method (line 10).